common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -108,6 +117,16 @@
      */
     private static final int RSA_KEY_SIZE = 2048;
 
+    /**
+     * Our designated signing algorithm; used for JWTs and generalized signing actions.
+     */
+    private static final String SHA_256_WITH_RSA = "SHA256withRSA";
+
+    /**
+     * Log message when private key material cannot be found.
+     */
+    public static final String PRIVATE_KEY_NOT_FOUND = "Not an instance of a PrivateKeyEntry";
```

Comment:
Why is this constant public when the other is not?

Reply:
Auto-refactor tool's default vis -- fixed in b39b87a82690fac2528ad8990973016a1c453516